addons/binding/org.openhab.binding.loxone/OSGI-INF/LoxoneHandlerFactory.xml

```diff
@@ -12,9 +12,14 @@
 <scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" immediate="true" name="org.openhab.binding.loxone.internal.LoxoneHandlerFactory">
 
    <implementation class="org.openhab.binding.loxone.internal.LoxoneHandlerFactory"/>
-   <reference bind="setChannelTypeProvider" cardinality="1..1" interface="org.eclipse.smarthome.core.thing.type.ChannelTypeProvider" name="ChannelTypeProvider" policy="dynamic" unbind="unbindChannelTypeProvider"/>
    <service>
       <provide interface="org.eclipse.smarthome.core.thing.binding.ThingHandlerFactory"/>
    </service>
+   <reference cardinality="1..1" 
+    bind="setChannelTypeProvider" 
+    interface="org.openhab.binding.loxone.internal.LoxoneChannelTypeProvider"
+    name="ChannelTypeProvider" 
+    policy="dynamic" 
+    unbind="unbindChannelTypeProvider"/>
```

Comment:
You should stay symmetric: as your bind method is called `setChannelTypeProvider`, your unbind should be `unsetChannelTypeProvider`.

Btw, you might want to consider to switch to DS annotations now - they are so much nicer and you don't have to deal with the XMLs yourself. See e.g. https://github.com/eclipse/smarthome/pull/4068/files.